recipe-server/client/control/utils/apiFetch.js

```diff
@@ -35,13 +35,6 @@ export default function apiFetch(url, options = {}) {
     delete settings.data;
   }
 
-  return fetch(`${API_ROOT}${url}${queryString}`, settings)
-    .then(response => Promise.all([response.ok, response.json()]))
-    .catch(error => [false, { message: error.message }])
-    .then(([ok, data]) => {
-      if (!ok) {
-        throw data;
-      }
-      return data;
-    });
+  const response = await fetch(`${API_ROOT}${url}${queryString}`, settings);
+  return response.json();
```

Comment:
Technically, the `await fetch(...)` business should be wrapped in a `try/catch` block to handle errors.